common/src/gas_provider/mod.rs

```diff
@@ -32,6 +34,8 @@ mod reservable;
 #[cfg(test)]
 mod property_tests;
 
+#[cfg(feature = "std")]
+pub use auxiliary::*;
```

Comment:
I think it worth to not re-export because of simple type names like `TotalIssuanceWrap`, `ExternalOrigin`, etc

Reply:
done in [00dec79](https://github.com/gear-tech/gear/pull/3962/commits/00dec791abb2807adb270767548d3fcec83eb50a)